IOMC/EventVertexGenerators/python/VtxSmearedRealistic2023PbPbCollision_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
```

Comment:
I think a new file should be created rather than editing the `2023PbPb` file to contain `2025OO`. In fact, the above modification to `Configuration/StandardSequences/python/VtxSmeared.py` will not run properly unless this is done.